var/spack/repos/builtin/packages/occa/package.py

```diff
@@ -50,13 +51,16 @@ class Occa(Package):
             description='Activates support for OpenCL')
     variant('coi',   default=False,
             description='Activates support for COI')
-    depends_on('cuda', when='+cuda')
-    
+    # depends_on('cuda', when='+cuda')
+
     def install(self, spec, prefix):
         os.environ['OCCA_CXX'] = self.compiler.cxx
 	# TODO: How can I get all the Cxx flags that Spack is using?
 	#os.environ['OCCA_CXXFLAGS'] =
-        os.environ['OCCA_CUDA_COMPILER'] = os.path.join(spec['cuda'].prefix,'bin','nvcc')
+        if (sys.platform == 'darwin'):
+            os.environ['OCCA_CUDA_COMPILER'] = '/usr/local/cuda/bin/nvcc'
+        else:
+            os.environ['OCCA_CUDA_COMPILER'] = os.path.join(spec['cuda'].prefix,'bin','nvcc')
```

Comment:
True, made that change after it worked on Linux, and before it worked on the Mac